src/components/Picklist/__test__/picklist.spec.js

```diff
@@ -48,7 +58,497 @@ describe('<Picklist />', () => {
             .instance()
             .activeChildren.findIndex(child => child.name === 'option1');
         expect(optionUnregisterChildFn).toHaveBeenCalled();
-        expect(component.instance().activeChildren.length).toEqual(2);
-        expect(itemIndex).toEqual(-1);
+        expect(component.instance().activeChildren.length).toBe(2);
+        expect(itemIndex).toBe(-1);
+    });
+    it('should open menu when click the picklist input', () => {
+        const component = mount(
+            <Picklist label="Picklist">
+                <PicklistOption label="Option 1" name="option1" />
+                <PicklistOption label="Option 2" name="option2" />
+                <PicklistOption label="Option 3" name="option3" />
+            </Picklist>,
+        );
+        jest.runAllTimers();
```

Comment:
I think this is not needed here
it is only used to test an async call with setTimeout